### DISCUSSION TITLE
dummy modulos

### DISCUSSION THREAD

**Discussion by Jingfu2000** [Ulm-IQO/qudi-iqo-modules#143] [General]
I am learning qudi by running the dummy modulos.  In  time_series_gui, e.g.,  in principle, we can save the data.  Where does the software save the data?  Thanks.

**Comment by TobiasSpohn** [comment]
A lot of qudi modules (e.g the time series toolchain) use the `module_default_data_dir` property as a save directory. You can retrieve its path by typing in the console `time_series_reader_logic.module_default_data_dir`. 
In general you can get the default data directory by typing in the console:
`from qudi.util import paths`
and
`paths.get_default_data_dir()`.
By using the `paths` utility you can also retrieve other important directory paths as e.g. the log directory.

**Comment by Jingfu2000** [comment]
Hello Tobias,
thanks a lot for your help.  I got the data, and re-plotted, see the fig.  Please check and comment.  

![image](https://github.com/user-attachments/assets/c4a5c190-8dbb-4503-b680-9f41746e8e78)
